src/material-experimental/mat-feature-highlight/feature-highlight-container.scss

```diff
@@ -0,0 +1,129 @@
+$open-animation-time: 350ms;
+$close-animation-time: 200ms;
+
+@keyframes feature-highlight-radial-pulse-expand {
+  0%, 33.333333% {
+    opacity: .54;
+    transform: scale(1, 1);
+  }
+
+  66.666666%, 100% {
+    opacity: 0;
+    transform: scale(2, 2);
+  }
+}
+
+@keyframes feature-highlight-inner-circle-expand {
+  0% {
+    transform: scale(1, 1);
+  }
+
+  33.333333% {
+    transform: scale(1.1, 1.1);
+  }
+
+  66.666666%, 100% {
+    transform: scale(1, 1);
+  }
+}
+
+:host {
+  display: block;
+  position: relative;
+}
+
+.feature-highlight-container-div {
+  position: absolute;
+  outline: none;
+}
+
+.feature-highlight-inner-circle {
+  cursor: pointer;
```

Comment:
Could we use a `button` for this so we don't have to style a `div` as if it's clickable?